roles/tpa_single_node/templates/manifests/guac/vexination-collector/Deployment.yaml.j2

```diff
@@ -35,47 +36,39 @@ spec:
             - collect
             - s3
             - --poll
-          env:
+          env: 
             - name: STORAGE_ACCESS_KEY
-              valueFrom:
-                secretKeyRef:
-                  key: root-user
-                  name: infrastructure-minio
+              value: {{ tpa_single_node_storage_access_key }}
             - name: STORAGE_SECRET_KEY
-              valueFrom:
-                secretKeyRef:
-                  key: root-password
-                  name: infrastructure-minio
+              value: {{ tpa_single_node_storage_secret_key }}
             - name: GUAC_S3_URL
-              value: http://infrastructure-minio:9000
+              value: {{ tpa_single_node_storage_endpoint }}
             - name: GUAC_S3_BUCKET
-              value: vexination-default
+              value: {{ tpa_single_node_storage_vexination_bucket }}
             - name: GUAC_S3_MP
-              value: kafka
+              value: {{ tpa_single_node_event_bus_type}}
             - name: GUAC_S3_MP_ENDPOINT
```

Comment:
GUAC_S3_MP_ENDPOINT and other Kafka fields are not mandate for AWS SQS, it can be wrapped with conditional statement. Reference [here](https://github.com/trustification/trustification-ansible/blob/main/roles/tpa_single_node/templates/manifests/v11y/indexer/Deployment.yaml.j2#L108)

Reply:
Thanks I missed that.